Example/healthvault-ios-sdk/ViewControllers/ActionPlanTasks/MHVActionPlanTaskListViewController.m

```diff
@@ -64,7 +64,7 @@ - (IBAction)addTask:(id)sender
 
     MHVActionPlanFrequencyTaskCompletionMetrics *metrics = [[MHVActionPlanFrequencyTaskCompletionMetrics alloc] init];
     metrics.occurrenceCount = @(1);
-    metrics.windowType = @"Daily";
+    metrics.windowType = MHVActionPlanFrequencyTaskCompletionMetricsWindowTypeEnum.Daily;
```

Comment:
...Invalidating caches and naming things; the two most difficult things in computer science.